### ISSUE TITLE
Event handlers do not update during hook lifetime

### ISSUE THREAD

**Issue by kajetan-intive** [NoriginMedia/Norigin-Spatial-Navigation#2]
**Describe the bug**
Event handlers are not updated. Only initial handlers are passed do `Focusable`

**To Reproduce**
1. Modify in example a function `onAssetPress`
```tsx
const onAssetPress = (asset) => {
  console.log(selectedAsset);
  setSelectedAsset(asset);
};
```
3. Press `Enter` in any of assets
4. In console see that no matter what current value of `selectedAsset` is, in console it outputs `null` all the time

**Expected behavior**
In console we should see proper current value from `selectedAsset`

**Additional context**
`onEnter` passed to `SpatialNavigation.addFocusable`
https://github.com/NoriginMedia/Norigin-Spatial-Navigation/blob/main/src/useFocusable.ts#L132
but when it should update there are no handlers passed to `SpatialNavigation.updateFocusable`
https://github.com/NoriginMedia/Norigin-Spatial-Navigation/blob/main/src/useFocusable.ts#L157

**Comment by clynchmerapar** [comment]
I think we may be experiencing the same [issue](#3) 😅

**Comment by asgvard** [comment]
This is fixed in 1.0.3 👍
